<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launcher</class>
 <widget class="QMainWindow" name="launcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>100</number>
    </property>
    <property name="rightMargin">
     <number>100</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <item>
       <widget class="QLabel" name="lbl_physics">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_time_freq">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Time-Frequency Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_wavelet_phase">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Wavelet Phase Coherence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ridge_extraction">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Ridge Extraction and Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_wavlet_bispectrum">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Wavelet Bispectrum Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dynamical_bayesian">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Dynamical Bayesian Interference</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
